samples/web-java-spring-petclinic/devfile.yml

```diff
@@ -0,0 +1,38 @@
+specVersion: 0.0.1
+name: spring-petclinic
+projects:
+  - name: spring-petclinic
+    source:
+      type: git
+      location: 'https://github.com/che-samples/web-java-spring-petclinic.git'
+tools:
+  - name: theia-ide
+    type: cheEditor
+    id: 'org.eclipse.che.editor.theia:1.0.0'
+  - name: exec-plugin
+    type: chePlugin
+    id: 'che-machine-exec-plugin:0.0.1'
+  - name: dev
+    type: dockerimage
+    image: registry.devshift.net/che/centos_jdk8
+    mountSources: true
+    endpoints:
+      - name: petclinic-web-server
+        port: 8080
+        attributes:
+          protocol: http
+          secure: 'false'
+          public: 'true'
+          discoverable: 'false'
+    memoryLimit: 2048M
+commands:
+  - name: build_&_run
+    actions:
+      - type: exec
+        tool: dev
+        command: "cd /projects/spring-petclinic && mvn clean install && cp /projects/spring-petclinic/target/*.war $TOMCAT_HOME/webapps/ROOT.war && $TOMCAT_HOME/bin/catalina.sh run 2>&1"
```

Comment:
Maybe it would be better to use `${CHE_PROJECTS_ROOT` (see https://github.com/eclipse/che/blob/07263f1e30089689d71b057f747a44a29283e3c4/wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/spi/provision/env/ProjectsRootEnvVariableProvider.java#L27) instead of hardcoded `/projects`.

Reply:
Maybe. But i doubt that end-users may know anything what about this variable. This will less clear than the direct path.

Reply:
But this variable is described in devfile JSON Schema https://github.com/eclipse/che/blob/master/wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json#L239
@metlos WDYT?

Reply:
@l0rd  wdyt ?

Reply:
+1 to use `${${CHE_PROJECTS_ROOT}`